### PR TITLE
feat added onboarding page

### PR DESCRIPTION
**Pull request by deepak29-git**
features:
1.add name
2.show greetings and time
3 add your main focus 

Screenshot (141)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+import './App.css';
+import { OnboardingPage } from './Pages/OnboardingPage/OnbordingPage';
```
Comment (VedanthB) [inline src/App.js:3]: you can do a barrel import / export here

File: src/Pages/OnboardingPage/OnbordingPage.jsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
+import "../OnboardingPage/OnboardingPage.css";
+import nextIcon from "../../assets/next.png";
+import { useEffect, useState } from "react";
+import { greetings } from "../../Utils/greetings";
+import { getMinuteBelowTen } from "../../Utils/getMinuteBelowTen";
```
Comment (VedanthB) [inline src/Pages/OnboardingPage/OnbordingPage.jsx:5]: barrel import / export , code looks cleaner

**Review by VedanthB — APPROVED** [review]
Good work deepak,

things i liked

- clean code
- good variable naming
you can merge the PR 🚀

**Review by Mohitkumar33 — APPROVED** [review]
Nice variable naming

File: src/Pages/OnboardingPage/OnbordingPage.jsx [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+  const continueHandler = () => {
+    setShow(true);
+    setHideMainFocusTitle(true);
+    setMainFocusInput("");
+  };
+
+  useEffect(() => {
+  
+  }, []);
```
Comment (ragini07) [inline src/Pages/OnboardingPage/OnbordingPage.jsx:26]: remove if not used

File: src/Pages/OnboardingPage/OnbordingPage.jsx (hunk 2 above)
Comment (deepak29-git) [reply src/Pages/OnboardingPage/OnbordingPage.jsx:5]: ok i will do it now

**Review by ragini07 — COMMENTED** [review]
Code looks good
Naming convention self explanatory
Suggestion- can use barrel export from utils
Also when you created react app there are some files in src which are of no use , so you can delete them permanently.Like - app.test.js, logo.svg, reportWebVitals.js

**Review by ragini07 — APPROVED** [review]

File: src/Pages/OnboardingPage/OnbordingPage.jsx (hunk 3 above)
Comment (deepak29-git) [reply src/Pages/OnboardingPage/OnbordingPage.jsx:26]: ok

File: src/App.js (hunk 1 above)
Comment (deepak29-git) [reply src/App.js:3]: done

**Comment by deepak29-git** [comment]
> Code looks good Naming convention self explanatory Suggestion- can use barrel export from utils Also when you created react app there are some files in src which are of no use , so you can delete them permanently.Like - app.test.js, logo.svg, reportWebVitals.js

done all the things

**Pull request merged**